packages/providers/onekey-bfc-provider/src/ProviderBfcBase.ts

```
@@ -1,16 +1,17 @@
 import { IInjectedProviderNames } from '@onekeyfe/cross-inpage-provider-types';
+
 import { ProviderBase, IInpageProviderConfig } from '@onekeyfe/cross-inpage-provider-core';
 
-class ProviderPrivateExternalAccountBase extends ProviderBase {
+class ProviderBfcBase extends ProviderBase {
   constructor(props: IInpageProviderConfig) {
     super(props);
   }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)